DSCResources/MSFT_xSQLServerSetup/MSFT_xSQLServerSetup.psm1

```diff
@@ -121,6 +121,8 @@ function Get-TargetResource
             $ASServer.Connect("localhost\$InstanceName")
         }
         $ASCollation = ($ASServer.ServerProperties | Where-Object {$_.Name -eq "CollationName"}).Value
+        $ASSERVERMODE =  $(IF ($($ASServer.ServerMode) -eq 'Default') {'MULTIDIMENSIONAL'}
```

Comment:
Variable names should be camel cased.  [Style Guide - variable names should be camel cased](https://github.com/PowerShell/DscResources/blob/TravisEz13-patch-1/StyleGuidelines.md#variable-names-should-use-camelcase)